.buildkite/release-automation/verify-macos-wheels.sh

```diff
@@ -16,6 +17,26 @@ fi
 mac_architecture=$1 # First argument is the architecture of the machine, e.g. x86_64, arm64
 export RAY_VERSION="${RAY_VERSION:-2.9.1}"
 export RAY_HASH="${RAY_HASH:-cfbf98c315cfb2710c56039a3c96477d196de049}"
+export USE_BAZEL_VERSION="${USE_BAZEL_VERSION:-5.4.1}"
+
+install_bazel() {
+    mkdir -p "$TMP_DIR/bin"
+    # Add bazel to the path.
+    export PATH="$TMP_DIR/bin:$PATH"
```

Comment:
this PATH thing should run before running the scripts? or right after making the temp dir. this is not really part of `install_bazel`.

Reply:
fixed